references/proxy-template/template-v1/apiproxy/proxies/default.xml

```diff
@@ -75,6 +76,14 @@
 
     <!-- insert your flows here and check scopes -->
 
+    <Flow name="GET /headers">
```

Comment:
The `/header` might be confusing in a generic usage of the generator.

How about just using `request.verb = "GET" and proxy.pathsuffix MatchesPath "/**"`?

Reply:
Modification: the flow name and condition are dynamically set based on the the path section of TARGET_URL.
The default value of TARGET_URL is https://httpbin.org/get